### DISCUSSION TITLE
Add project_url to defaultTemplateData in MailService

### DISCUSSION THREAD

**Discussion by ybelenko** [directus/directus#11856] [Feature Requests]
https://github.com/directus/directus/blob/0bad2744b39ddb4ccbbee867855dc5a9d812d4c8/api/src/services/mail/index.ts#L95

`projectUrl` might be useful in custom email footer. 

```html
<footer>
  This letter has been sent from our 
  <nav>
    <a href="{{ projectUrl }}">Awesome Project</a>
  </nav>
</footer>
```

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
